.env_SAMPLE

```diff
@@ -177,7 +178,8 @@ export WATCHMAN_TOKENS=''
 # Project configuration - set up working state.
 ###############################################
 
-if [ ! -z $(command -v workon) ]; then
+# Local envvar overrides and virtualenv hook
+if [ $(uname -s) = "Darwin" ]; then
```

Comment:
Could you explain this change just a little more for me? This will only be true on macOS, right? What if someone wants to develop on a Linux system? Why was the previous condition less good?